ibm_catalog.json

```diff
@@ -9,14 +9,14 @@
         "target_terraform",
         "terraform",
         "solution",
-        "containers",
-        "converged_infra"
+        "containers"
       ],
       "keywords": [
         "terraform",
         "ocp",
         "cluster",
-        "red_hat_openshift"
+        "red_hat_openshift",
+        "ROKS"
```

Comment:
I dont know if this is case sensitive? Perhaps check with catalog team? should we use "roks" ?